src/forms/CurrentExpenses.js

```diff
@@ -348,12 +363,16 @@ const ExpensesFormContent = function ({ current, time, setClientProperty }) {
             { ...sharedProps }
             generic={ 'disabledAssistance' }> Disabled/Handicapped assistance 
           </CashFlowRow>
-          <CashFlowRowAfterConfirm
-            { ...sharedProps }
-            generic={ 'earnedBecauseOfAdultCare' }
-            confirmLabel={ 'Do assistance expenses allow you to make additional income?' }>
-            <span style={{ textDecoration: 'underline' }}>Income</span> made possible by assistance expenses
-          </CashFlowRowAfterConfirm>
+
+          <EarnedFrom
+            heading={ 'Effect of Assistance' }
+            cashflowProps={{
+              ...sharedProps,
+              generic:      'earnedBecauseOfAdultCare',
+              confirmLabel: 'If you didn\'t have that assistance, would it affect your current job?',
+            }}>
+            How much less?
```

Comment:
I think we should change this to "How much less would you make?" for consistency.

Reply:
Oh, shoot. Definitely. I just forgot to update that.